smithy-model/src/main/java/software/amazon/smithy/model/validation/NodeValidationVisitor.java

```diff
@@ -388,6 +391,32 @@ public List<ValidationEvent> memberShape(MemberShape shape) {
         return events;
     }
 
+    public List<ValidationEvent> checkNullMember(MemberShape shape) {
+        List<ValidationEvent> events = new ArrayList<>();
```

Comment:
```suggestion
```

Note that the switch can have its `break;` lines removed as well, since each case will have a `return`.